ITE/9/README.adoc

```diff
@@ -45,7 +45,7 @@ The link:https://github.com/in-toto/attestation/[in-toto Attestation Framework]
 
 When discussing a new attestation type, what is really being defined is a new type of link:https://github.com/in-toto/attestation/tree/main/spec#predicate[predicate]. The developer(s) of this new predicate must create a formal document that specifies the predicate type, and discusses several other related aspects. The purpose and prerequisites for the new attestation type MUST be defined. The document should also include a model that describes how the various components of the predicate map to supply chain semantics, juxtaposed with detailed use cases. These use cases SHOULD include example metadata that clearly highlight how the proposed predicate addresses the shortcoming described by them. Including these aspects will aid readers and future adopters understand the why and how of a particular attestation type.
 
-The specification MUST contain a detailed schema of the fields that make up the predicate. This schema MUST define unambiguously a unique `predicateType` following the rules defined by the attestation framework. The schema must also clearly define the types of every field, with ranges for values where applicable. Further, the specification MUST contain rules for parsing the fields, as well as make recommendations for handling edge cases, where parsing or handling of fields would deviate from the standard link:https://github.com/in-toto/attestation/tree/main/spec/v1.0-draft#parsing-rules[parsing rules]. For example, it may be preferable in some cases to silently allow extra, undefined fields, while in other cases such metadata should be rejected.
+The specification MUST contain a detailed schema of the fields that make up the predicate. This schema MUST define unambiguously a unique `predicateType` and a shorter `predicateName` following the rules defined by the attestation framework. The `predicateName` must be used as the predicate specification's file name. The name must not include version information and is expected to only be used as an unauthenticated hint in other contexts such as a media type to indicate the predicate stored in an attestation. The schema must also clearly define the types of every field, with ranges for values where applicable. Further, the specification MUST contain rules for parsing the fields, as well as make recommendations for handling edge cases, where parsing or handling of fields would deviate from the standard link:https://github.com/in-toto/attestation/tree/v1.0/spec/v1.0#parsing-rules[parsing rules]. For example, it may be preferable in some cases to silently allow extra, undefined fields, while in other cases such metadata should be rejected.
```

Comment:
Nit: Can you please wrap at 80 chars?

Reply:
What may be misleading here is the term "schema" in the second sentence. Both the predicateType and predicate name aren't meant to be included in the predicate schema itself. Suggesting a few wording changes...

```suggestion
The specification MUST define unambiguously a unique `predicateType`, a shorter `predicateName` following the rules defined by the attestation framework, and contain a detailed schema of the fields that make up the predicate. Any files related to the predicate specification (spec document, directories, protobuf definition) MUST match the `predicateName`. The name must not include version information and is expected to only be used as an unauthenticated hint in other contexts such as a media type to indicate the predicate stored in an attestation. The schema must also clearly define the types of every field, with ranges for values where applicable. Further, the specification MUST contain rules for parsing the fields, as well as make recommendations for handling edge cases, where parsing or handling of fields would deviate from the standard link:https://github.com/in-toto/attestation/tree/v1.0/spec/v1.0#parsing-rules[parsing rules]. For example, it may be preferable in some cases to silently allow extra, undefined fields, while in other cases such metadata should be rejected.
```

Reply:
One more comment: It may be helpful to provide an example for how `predicateName` should be used in file naming.

Reply:
> Nit: Can you please wrap at 80 chars?

happy to wrap this paragraph. For some reason, I don't seem to have submitted ITE-9 originally and I wasn't sure about making out-of-context changes.

Reply:
I applied your suggestion. I also added an example for the name.